src/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
-    <DebugType>portable</DebugType>
+    <!--<DebugType>portable</DebugType>-->
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="SourceLink.Create.CommandLine" Version="2.1.0" PrivateAssets="all" />
+    <!--<PackageReference Include="SourceLink.Create.CommandLine" Version="2.1.0" PrivateAssets="all" />-->
```

Comment:
Debugging does work for me, although I never found debugging particularly useful for working on this function. Way too many iterations to go through, but maybe I should have tried conditional breakpoints.